Simple Inventory/Inventory.cs

```diff
@@ -1,65 +1,43 @@
-﻿
-namespace SimpleInventory
-{
-       class Inventory
-       {
-            private Dictionary<int, Product> products = new Dictionary<int, Product>();
-            public IReadOnlyDictionary<int, Product> Products => products;
+﻿using Simple_Inventory.Interfaces;
 
-            public Inventory()
-                {
-                    products = new Dictionary<int, Product>();
-                }
+namespace SimpleInventory;
 
-            public bool AddProduct(Product product)
-            {
-                if (products.ContainsKey(product.ID))
-                {
-                    return false;
-                }
-                else
-                {
-                    products.Add(product.ID, product);
-                    return true;
-                }
-            }
+class Inventory
+{
+    private IDataBase DataBase;
```

Comment:
```suggestion
    private readonly IDataBase _dataBase;
```

or you can even use primary constructor here.